app/models/share/email.rb

```diff
@@ -17,9 +17,9 @@ class Share::Email < ActiveRecord::Base
   include Share::Variant
 
   validates :subject, :body, presence: true
-  validate :has_link
+  validate :has_link, unless: -> { body.nil? }
```

Comment:
is it safe to remove this? This change makes the validation run even when the body is nil.